examples/multi_label/classifier_chain.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
```

Comment:
I was about to suggest the confusion matrix, but I think we haven't implemented the multi-label confusion matrix yet.
It's actually not really important to add a plot for it to be a ``plot_`` example. That just ensures it runs. But it requires that the example runs quickly enough. If it doesn't, maybe try yeast instead?